lib/oban/plugins/cron.ex

```diff
@@ -107,9 +126,21 @@ defmodule Oban.Plugins.Cron do
   def handle_info(:evaluate, %State{} = state) do
     state = schedule_evaluate(state)
 
-    Query.with_xact_lock(state.conf, state.lock_key, fn ->
-      insert_jobs(state.conf, state.crontab, state.timezone)
-    end)
+    start_metadata = %{config: state.conf, plugin: __MODULE__}
+
+    :telemetry.span(
+      [:oban, :plugin],
+      start_metadata,
+      fn ->
+        case lock_and_insert_jobs(state) do
+          {:ok, inserted_jobs} when is_list(inserted_jobs) ->
+            {:ok, Map.put(start_metadata, :jobs, inserted_jobs)}
+
+          error ->
+            {:error, Map.put(start_metadata, :error, error)}
+        end
+      end
+    )
```

Comment:
This has the same `{:ok, false}` response due to the use of xact lock. I think we can save an extra three lines of vertical space rather cleanly:

```suggestion
    :telemetry.span([:oban, :plugin], start_metadata, fn ->
      case lock_and_insert_jobs(state) do
        {:ok, inserted_jobs} when is_list(inserted_jobs) ->
          {:ok, Map.put(start_metadata, :jobs, inserted_jobs)}

        {:ok, false} when is_list(inserted_jobs) ->
          {:ok, Map.put(start_metadata, :jobs, [])}

        error ->
          {:error, Map.put(start_metadata, :error, error)}
      end
    end)
```